esrally/utils/git.py

```diff
@@ -67,19 +67,19 @@ def is_branch(src_dir, identifier):
 def clone(src, *, remote):
     io.ensure_dir(src)
     # Don't swallow subprocess output, user might need to enter credentials...
-    if process.run_subprocess_with_logging("git clone %s %s" % (remote, io.escape_path(src))):
+    if process.run_subprocess_with_logging("git clone %s %s" % (remote, io.escape_path(src)), level=logging.DEBUG):
```

Comment:
I think we should keep all of the git module logging at `INFO`. These commands are only invoked as a benchmark is starting, so their volume shouldn't be a concern, and they're generally useful in verifying that the correct actions took place (such as for building from source, teams, etc.).

Side note: we'll likely have some conflicts due to https://github.com/elastic/rally/pull/1602

Reply:
Yeah, this was one I was on the fence about. I'm fine with keeping at INFO, so I'll adjust. Thanks for the feedback!